### ISSUE TITLE
Inquiry on results in the original paper

### ISSUE THREAD

**Issue by Derekkk** [lanwuwei/SPM_toolkit#9]
Hi Wuwei, 

Thanks for sharing the code. I have a question on the results in Table 4 of the original paper. Are those results(let's say acc of PWIM on Quora is 0.834) on training set or test set? I noticed that the results in Table 4 are quite corresponding to the results of the training curves in Figure 3, so just want to make sure. Thank you!

**Comment by lanwuwei** [comment]
Hi Derek, 
These results (Table 4) are for test set, since each dataset has its own training/test set(Table 2). I trained the model on training part and tested it on the corresponding test part. As for Quora, I followed previous work to do training/test split: (Zhiguo Wang's partition)[https://zhiguowang.github.io/]
